snuba/query/processors/timeseries_column_processor.py

```diff
@@ -0,0 +1,69 @@
+from typing import Mapping, Optional
+
+from snuba.query.dsl import multiply
+from snuba.query.expressions import Column, Expression, FunctionCall, Literal
+from snuba.query.query import Query
+from snuba.query.query_processor import QueryProcessor
+from snuba.request.request_settings import RequestSettings
+
+
+class TimeSeriesColumnProcessor(QueryProcessor):
+    """
+    Mimics the old column_expr functionality
```

Comment:
Since this will be the only code executing the logic that today is in column_expr, could you please actually describe what this is for in the docstring as if this was the only place this transformation was defined ?